### ISSUE TITLE
Take switch out of checkbox

### ISSUE THREAD

**Issue by tomdye** [dojo/widgets#796]
This issue is concerned with removing the switch `mode` from checkbox.
We should create a separate switch widget and ensure that it is both theme-able and accessible.
It should be based on a checkbox input to ensure a11y compliance.

**Comment by ericos** [comment]
Had some time earlier this week to work on this widget and Dojo/Material themes. Will update tests and get in a PR this week.

https://github.com/ericos/widgets/tree/separate-switch-widget
